Library/Homebrew/cmd/test-bot.rb

```diff
@@ -675,7 +676,8 @@ def cleanup_after
         test "git", "clean", "-ffdx"
         HOMEBREW_REPOSITORY.cd do
           safe_system "git", "reset", "--hard"
-          safe_system "git", "clean", "-ffdx", "--exclude=/Library/Taps/"
+          Tap.names.reject { |t| t == "homebrew/core" }.each { |s| safe_system "brew untap #{s}" }
+          safe_system "git", "clean", "-ffdx"
```

Comment:
Similar story like in the other two comments, only this time we probably don't mind wiping **all taps**, though retaining `homebrew/core` still seems desirable.